cartridge/failover.lua

```diff
@@ -730,6 +804,57 @@ local function cfg(clusterwide_config)
             end,
         })
         vars.failover_fiber:name('cartridge.stateful-failover')
+
+    elseif failover_cfg.mode == 'raft' then
+        box.cfg{
+            replication_connect_quorum = 0,
+            -- The instance is set to candidate, so it may become leader itself
+            -- as well as vote for other instances.
+            --
+            -- Alternative: set one of the three instances to `voter`, so that it
+            -- never becomes a leader but still votes for one of its peers and helps
+            -- it reach election quorum (2 in our case).
+            election_mode = os.getenv('TARANTOOL_ELECTION_MODE') or 'candidate',
+            -- Quorum for both synchronous transactions and
+            -- leader election votes.
+            replication_synchro_quorum = failover_cfg.replication_synchro_quorum or 2,
+            -- Synchronous replication timeout. The transaction will be
+            -- rolled back if no quorum is achieved during 1 second.
+            replication_synchro_timeout = 1,
+            -- Heartbeat timeout. A leader is considered dead if it doesn't
+            -- send heartbeats for 4 * replication_timeout (1 second in our case).
+            -- Once the leader is dead, remaining instances start a new election round.
+            replication_timeout = 0.25,
+            -- Timeout between elections. Needed to restart elections when no leader
+            -- emerges soon enough.
+            election_timeout = 0.25, -- = 4 * repl_timeout
+        }
+
+        local election = box.info.election
+
+        vars.raft_term = election.term
+
+        local leader = box.info.replication[election.leader] or {}
```

Comment:
Leader won't have time to elect here, so it'll be `{}`.

Is it possible to yield here until leader is found?